### ISSUE TITLE
Chromium Renderer process just crashed on Jenkins

### ISSUE THREAD

**Issue by omelnik** [cypress-io/cypress#9036]
Running into this issue on Jenkins. I've been running Cypress for a month already and didn't have a problem. Once I updated the version from `5.3.0` -> `5.5.0` started having this issue. 

NOTE: My tests are not in a docker container.

```
We detected that the Chromium Renderer process just crashed.

This is the equivalent to seeing the 'sad face' when Chrome dies.

This can happen for a number of different reasons:

- You wrote an endless loop and you must fix your own code
- There is a memory leak in Cypress (unlikely but possible)
- You are running Docker (there is an easy fix for this: see link below)
- You are running lots of tests on a memory intense application
- You are running in a memory starved VM environment
- There are problems with your GPU / GPU drivers
- There are browser bugs in Chromium

You can learn more including how to fix Docker here:

https://on.cypress.io/renderer-process-crashed
```

```
┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Cypress:      5.5.0                                                                            │
  │ Browser:      Electron 85 (headless)                                                           │
  │ Specs:        2 found (webapp/authenctication.js, webapp/booking.js)                           │
  │ Searched:     cypress/specs/webapp/authenctication.js, cypress/specs/webapp/booking.js         │
  │ Experiments:  experimentalFetchPolyfill=true,experimentalNetworkStubbing=true                  │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘

```

**Comment by radDunin** [comment]
Hi, I've seen it before but often at docker in you're case maybe try run tests in headless mode, or limit memory usage in some ways at Jenkins. Also, you can change the browser for electron and try with it.

**Comment by jennifer-shehane** [comment]
You can log the memory and CPU usage of Cypress by following these instructions: https://on.cypress.io/troubleshooting#Log-memory-and-CPU-usage This can help determine if Cypress is running out of resources and whether the machine requires more.

**Comment by omelnik** [comment]
@jennifer-shehane thank you so much for the suggestion that's a really cool feature. BTW I rolled back to `cypress-3.3.0` and it works perfectly fine, but still doesn't work with `cypress-3.5.0`. any ideas why?

**Comment by omelnik** [comment]
I decided to put my tests into a docker container and it solved the issue with Chromium, but I'm facing a new issue now.
When I tried to map my current dir to a folder inside my container, I started getting this error: `Could not find a Cypress configuration file, exiting. We looked but did not find a cypress.json file in this folder: /usr/src/app`

Here's my Dockerfile:

```sh
# pull official Cypress image
FROM cypress/included:5.5.0

# set working directory
RUN mkdir /usr/src/app
WORKDIR /usr/src/app

# copy our test application
COPY package.json /usr/src/app/package.json
COPY package-lock.json /usr/src/app/package-lock.json

# install NPM dependencies and Cypress binary
RUN npm ci

# copy all files into the container
COPY . /usr/src/app
COPY cypress.json /usr/src/app/cypress.json

# check if the binary was installed successfully
RUN $(npm bin)/cypress verify

# run tests
CMD ["./node_modules/.bin/cypress", "run"]
````

Here's my command
```
docker run -v $PWD:/usr/src/app --ipc=host erlond:latest --headless --browser chrome
```

And like usually it works on my local, but doesn't work on Jenkins. I'm not sure what I'm missing. I was using [this article](https://www.cypress.io/blog/2019/05/02/run-cypress-with-a-single-docker-command/) as the reference. Just for the record, I tried to pass `-w /usr/src/app` flag as well.

**Comment by omelnik** [comment]
This issue can be closed as I discovered another solution for my problem.
